common/lib/xmodule/xmodule/services.py

```diff
@@ -0,0 +1,19 @@
+"""
+Module contains various XModule/XBlock services
```

Comment:
I don't fully understand how `XBLOCK_SETTINGS` is meant to be configured. In the example you gave in this PR you have
```
"XBLOCK_SETTINGS": {
    "mentoring": {
        "theme": {
            "package": "mentoring",
            "locations": ["public/themes/apros.css"]
        }
    }
},
```

but in tests it looks like you just have
```
XBLOCK_SETTINGS = {xblock_setting_key2: {'b': 1}}
```

In the first example it looks like the settings must be specified by block name, but in the test overrides I don't see that as a requirement.

Can you document more clearly in this docstring how the XBLOCK_SETTINGS are meant to be configured and used?